Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:lts as build-stage
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Install deps
+RUN apt-get update -y \
+    && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get install --no-install-recommends -y wget=1.* \
+    && rm -rf /var/lib/apt/lists/*
+# Download latest nvim stable Github release. See:
+# https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8
+RUN wget -O - --progress=dot:giga https://api.github.com/repos/neovim/neovim/releases/latest \
+    | grep 'browser_download_url.*nvim.appimage"' \
+    | cut -d : -f 2,3 \
+    | tr -d \" \
+    | wget --progress=dot:giga -i -
```

Comment:
I think it might be easier to just install curl and let it follow the redirections for us with `curl -L https://github.com/neovim/neovim/releases/download/nightly/nvim.appimage`. Feel free to open a new PR if you feel like implementing this improvements :)

Reply:
Yes, `wget` is not the best tool for single file downloads. If you are going to crawl and archive a whole site it is great, otherwise use `curl`. The `-O` flag is particularly useful to save with a name automatically, and combine with `-J` to prefer the name specified by the server rather than just deducing it from the URL. Hence `curl -LJO <url>` becomes a good way to download and save files as a replacement for `wget <url>` while also following redirects.

Reply:
Can you please clarify why is `curl` better than `wget` for this? `wget` follows redirects automatically.
BTW, I think you can improve the `curl` invocation by adding `-f`.

Reply:
One reason is that it is installed by default in more places so you don't have to jump through as many hoops to get it. Another reason is it handles SSL stuff better these days, `wget` is quite a bit slower to adapt. And yes `wget` can follow some redirects, but not as many different kinds. There are more reasons too, but this is all moot anyway as I don't think we need to manually download anything at all. See #1015 for a simplification of all this.

Reply:
@infokiller The reason I recommended curl is that I've had issues with wget and redirects in the past. But maybe the problem was between the chair and the keyboard ;).
I don't have any other reason to prefer one over the other.